src/components/App/App.tsx

```diff
@@ -84,6 +84,7 @@ class App extends React.Component<{}, AllData> {
                   <SortBox updateSort={this.updateSort} />
                   {this.state.error && <h2>{this.state.error}</h2>}
                   {!this.state.error && !this.state.allPlanets.length && <h2>Loading...</h2>}
+                  {!this.state.error && !!this.state.allPlanets.length && <h2>Click on a planet to learn more!</h2>}
```

Comment:
!! is weird but I glad you got it to work!

Reply:
^^^